lib/src/revset.rs

```diff
@@ -262,14 +262,34 @@ impl RevsetExpression {
         )))
     }
 
+    pub fn bookmarks(pattern: StringPattern) -> Rc<RevsetExpression> {
+        Rc::new(RevsetExpression::CommitRef(RevsetCommitRef::Branches(
+            pattern,
+        )))
+    }
```

Comment:
I think we can just rename `branches()` to `bookmarks()`. These are just internal functions, right?

Reply:
Makes sense, removed.